### PR TITLE
Setup dependabot config

### LINKED ISSUE #1230: Setup automatique dependencies update bot
Adding a dependencies update bot to the project has already been mentioned in the community channels. I would like to open a ticket in order to make a decision, and get this done.

The 2 bots I know of are:
- https://dependabot.com/
- https://github.com/renovatebot/renovate

Using dependabot is pretty straightforward, renovate seems to have more knobs we can configure.
I would go with dependabot, which is probably enough for this project needs.

Who can set this up ?

### PR DESCRIPTION
**Pull request by jooola**
Fixes #1230

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  #     interval: "weekly"
+
+  # Maintain dependencies for Python apps
+  - package-ecosystem: "pip"
+    directory: "/api"
+    schedule:
+      interval: "daily"
+
+  - package-ecosystem: "pip"
+    directory: "/python_apps/analyzer"
```
Comment (paddatrapper) [inline .github/dependabot.yml:24]: This should be pypo

**Comment by jooola** [comment]
Woups, yeah, I pushed this change a bit too fast. Fixed it.

**Comment by jooola** [comment]
Ok, we might expect a rain of Issues / PR coming soon.

**Comment by paddatrapper** [comment]
Is there anything I need to configure still, or will the boot detect the file and do its thing automatically?

**Comment by jooola** [comment]
No, I though you'd need to do anything, but that's it.

**Pull request merged**